test/e2e/framework/network/utils.go

```diff
@@ -1134,19 +1134,39 @@ func TestUnderTemporaryNetworkFailure(ctx context.Context, c clientset.Interface
 //	}
 func BlockNetwork(ctx context.Context, from string, to string) {
 	framework.Logf("block network traffic from %s to %s", from, to)
-	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump REJECT", to)
-	dropCmd := fmt.Sprintf("sudo iptables --insert %s", iptablesRule)
-	if result, err := e2essh.SSH(ctx, dropCmd, from, framework.TestContext.Provider); result.Code != 0 || err != nil {
-		e2essh.LogResult(result)
-		framework.Failf("Unexpected error: %v", err)
+	nftablesRule := fmt.Sprintf("OUTPUT ip daddr %s reject", to)
+	dropCmdNftablesRule := fmt.Sprintf("sudo nft add rule ip nat %s", nftablesRule)
```

Comment:
> nftables doesn't have guaranteed pre-existing tables and chains like iptables does.

Oh, you have a table named `nat` with a chain named `OUTPUT` because of iptables-nft; that's the translation of the iptables `OUTPUT` chain. But not all hosts will have that (and it's bad style in nftables to use someone else's table anyway).